packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -329,6 +330,10 @@ export async function importBlock(
   // This adds the state necessary to process the next block
   this.stateCache.add(postState);
 
+  if (!isStateValidatorsNodesPopulated(postState)) {
+    this.logger.verbose("After importBlock caching postState without SSZ cache", {slot: postState.slot});
```

Comment:
do we need any metrics for this?

Reply:
- This is already covered in the state transition metric set from https://github.com/ChainSafe/lodestar/pull/5171 not sure if a metric adds value. The purpose of adding this was meant more for debugging specific instances